packages/@webex/internal-plugin-llm/src/llm.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable consistent-return */
```

Comment:
for all the places where we disable a rule for a whole file like here - should we have a jira for each to fix these?

Reply:
added to the same jira ticket https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-396390